compose.yml

```diff
@@ -60,10 +62,10 @@ services:
         source: ${COUNTERPARTY_DOCKER_DATA:-~/counterparty-docker-data}/
         target: /data
     ports:
-      - "4000:4000"                           # mainnet
-      - "14000:14000"                         # testnet
+      - "4000:4000"                                        # mainnet
+      - "14000:14000"                                      # testnet
     command:
-      - "--chain=${NETWORK:-main}"            # export NETWORK=test for testnet
+      - "--chain=${COUNTERPARTY_NETWORK:-main}"            # export COUNTERPARTY_NETWORK=test for testnet
```

Comment:
hm, I prefer the old style of `--testnet`/`--regtest` etc. that's also what Ordinals does, fwiw.

unfortunately I think that bitcoind uses tons of antipatterns in its CLI and API :/

Reply:
The old style still works! you cas use --testnet/--regtest and that what we could document everywhere.
It's really just to have a really simple `compose.yml`.. It can't hurt to have an extra argument just for that imho..

Reply:
ok found a solution without the flag `--chain` ;-) see https://github.com/CounterpartyXCP/counterparty-core/pull/1548/commits/2b0c2d13b004282119e3d5bd8dd412ce5fba3841